tf-psa-crypto/drivers/builtin/src/ecp.c

```diff
@@ -2914,6 +2914,9 @@ int mbedtls_ecp_check_privkey(const mbedtls_ecp_group *grp,
         if (mbedtls_mpi_get_bit(d, 0) != 0 ||
             mbedtls_mpi_get_bit(d, 1) != 0 ||
             mbedtls_mpi_bitlen(d) - 1 != grp->nbits) {  /* mbedtls_mpi_bitlen is one-based! */
```

Comment:
We could just rewrite this and avoid getting bogged down by a subtraction.
```suggestion
            mbedtls_mpi_bitlen(d) != grp->nbits + 1) {  /* mbedtls_mpi_bitlen is one-based! */
```